lib/validation-stream.js

```diff
@@ -9,6 +9,12 @@ var Duplex = require('readable-stream').Duplex;
 
 var XmlParser = require('./xml-parser');
 
+/** @arg {?string} str Input string to check.
+    @return {!boolean} True if input is undefined, null, or entirely whitespace. */
```

Comment:
I'm not bashing the documentation itself. This was mostly a comment on keeping the code base consistent. If we want to document detecting nullity and such, then we could have added an inline comment:

```js
function isBlank(str) {
  // Returns true if input is undefined, null, or entirely whitespace
  return !str || ...;
}
```